frontend/src/App.test.tsx

```diff
@@ -152,6 +152,7 @@ describe("App", () => {
 
```

Comment:
One last thing -- I think it would be useful to add a unit test that essentially tests the same thing as the "main menu is hidden/shown" e2e tests.

The test coverage is a bit redundant given the e2e tests, but the scope of `App.tsx` unit tests is more precise since the e2e tests also test config option code.